spec/API_specification/signatures/elementwise_functions.py

```diff
@@ -1112,9 +1112,40 @@ def remainder(x1: array, x2: array, /) -> array:
     """
     Returns the remainder of division for each element ``x1_i`` of the input array ``x1`` and the respective element ``x2_i`` of the input array ``x2``.
 
+    .. note::
+       This function is equivalent to the Python modulus operator ``x1_i % x2_i``.
+
     .. note::
        For input arrays which promote to an integer data type, the result of division by zero is unspecified and thus implementation-defined.
 
+    **Special Cases**
```

Comment:
Very minor nitpick—we seem to be inconsistent in casing the "c" in case :sweat_smile:

Reply:
```suggestion
    **Special cases**
```